### DISCUSSION TITLE
Sailor Hat Daemon Reporting to Other Software

### DISCUSSION THREAD

**Discussion by dbeckwith01** [hatlabs/discussions#11] [Q&A]
Hi, I just received my Sailor Hat's last week and starting on a NMEA 2000 project with them.  I'm wondering if it's possible to have Raspberry Pi daemon report it's information to another app, including the voltage levels but also an alert that it will be powered down.  Sorry I'm new to Linux development.

Thanks,

**Comment by mairas** [comment]
Sure, it would be possible but just hasn't been implemented yet. One possible approach would be to make the daemon a client for Signal K and have it send Signal K deltas. I think one could even write a standalone script to do that - there might be some concurrency issues for several processes accessing the same I2C device. In practice I don't think it's likely to result in any significant problems. You can check out the `sh-rpi-i2c-report` script installed on the Pi together with the daemon.
